add2nos.c

```diff
@@ -0,0 +1,55 @@
+#include<iostream>
+using namespace std;
+
+int add1(int a, int b){
+  int c = a+b;
+  return c;
+}
+
+int add2(int a, int b){
+  return a+b;
+}
+int add6(int &a,int&b){
+  cout<<"refernce ";
+  return a+b;
+}
+int add7(int *a, int *b){
+  cout<<"pointer ";
+  return (*a) + (*b);
+}
+
+int& add3(int a, int b){
+  int c=a+b;
+  return c;
+}
+
+int* add4(int a, int b){
+  int c=a+b;
+  return &c;
+}
```

Comment:
Returning Local variables' ref,add ,why??

Reply:
Did you run your code before committing? or you did this on purpose?
Reply to me before 9 pm ist today

Reply:
bkms